blog/2025-11-11-cloudstack-4.22-release/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Apache CloudStack 4.22.0.0 Release
+tags: [announcement]
+authors: [harikrishna]
+slug: cloudstack-4.22.0.0-release
+---
+
+[![](4.22.png "Apache CloudStack 4.22.0.0 Release")](/blog/cloudstack-4.22.0.0-release)
+
+The Apache CloudStack project is pleased to announce the release of CloudStack 4.22.0.0, which is a LTS release.
+
+The CloudStack 4.22 release is the most recent release of the cloud management platform. It comes as a product of extensive contributions from the project development community.
+
+The 4.22 release contains 10 new features, around 15 improvements and more than 140 bug fixes since 4.21. Some of the highlights include:
+
+<!-- truncate -->
+
+· Enhanced Backup and Disaster Recovery
+
+· SSL Offloading for Load Balancers
+
+· Baremetal/MaaS Extension
+
+· CSI Driver for CKS
+
+· Console Access for Proxmox in Extensions Framework 
+
+· VMware-to-KVM Migration Enhancements
+
+· Snapshot/Backup Schedule Listing
+
+· Per-Zone Console Proxy Configuration
+
+· Direct Volume Migration within Cluster
+
+· Persistent KVM Domains
+
+· Support for userdata on System VMs
```

Comment:
```suggestion
· Support for User Data on System VMs
```

updated as per of polish pr here - https://github.com/apache/cloudstack/pull/10856